Kudu.Core/SourceControl/Git/LibGit2SharpRepository.cs

```diff
@@ -225,7 +227,7 @@ public void FetchWithoutConflict(string remoteUrl, string branchName)
                     using (tracer.Step("LibGit2SharpRepository Fetch"))
                     {
                         // This will only retrieve the "master"
-                        repo.Network.Fetch(remote);
+                        LibGit2Sharp.Commands.Fetch(repo, remote.Name, remote.RefSpecs.Select(r => r.Specification), new FetchOptions(), "LibGit2SharpRepository Fetch");
```

Comment:
That seems awfully complicated to do a simple fetch. Doc still claims that simple one works: https://github.com/libgit2/libgit2sharp/wiki/git-fetch. Maybe worth opening issue to ask clarification?